bind/flag.go

```diff
@@ -32,14 +33,20 @@ func ConfigFile(fs *pflag.FlagSet, configFile *string) {
 			"The following precedence order of configuration sources is used: command flags, environment variables, config file, default values. ")
 }
 
-func DNSConfig(fs *pflag.FlagSet, cfg *forwarder.DNSConfig) {
+func DNSConfig(fs *pflag.FlagSet, cfg *osdns.Config) {
 	fs.VarP(anyflag.NewSliceValue[netip.AddrPort](nil, &cfg.Servers, forwarder.ParseDNSAddress),
 		"dns-server", "n", "<ip>[:<port>]"+
 			"DNS server(s) to use instead of system default. "+
-			"If specified multiple times, the first one is used as primary server, the rest are used as fallbacks. "+
+			"If specified multiple times, the first one is used as primary server, the rest are used as fallbacks, "+
+			"unless the round-robin option is enabled. "+
 			"The port is optional, if not specified the default port is 53. ")
+
 	fs.DurationVar(&cfg.Timeout,
-		"dns-timeout", cfg.Timeout, "Timeout for dialing DNS servers.")
+		"dns-timeout", cfg.Timeout, "Timeout for dialing DNS servers. "+
+			"Only used if DNS servers are specified. ")
+
+	fs.BoolVar(&cfg.RoundRobin, "dns-round-robin", cfg.RoundRobin, "Enable round-robin DNS server selection. "+
+		"Won't have any effect if DNS servers are not specified. ")
```

Comment:
This sentence is adds no value. Add something more relevant

If more than one DNS server is specified with the --dns-server flag, passing this flag ....